zerver/views/messages.py

```diff
@@ -799,6 +808,7 @@ def update_message_backend(request, user_profile,
 @has_request_variables
 def json_fetch_raw_message(request, user_profile,
                            message_id=REQ(converter=to_non_negative_int)):
+    # type: (HttpRequest, UserProfile, str) -> HttpResponse
```

Comment:
that `converter=to_non_negative_int` means `message_id` is an `int` in the body of this function